pom.xml

```diff
@@ -79,6 +79,7 @@
 		<module>utils/serde</module>
 		<module>distro</module>
 		<module>docs</module>
+		<module>tests</module>
```

Comment:
I have removed the `tests` module from the default list of modules because the tests do not run on Windows.  Instead, there is now a **profile** called `tests` that you can enable to run the tests.  I think that's a better solution.  So if you could remove this line that'd be good.

Reply:
i have added profiles, and by default our tests are not executed. Also i edited readme to execute them all.